Traditional Chinese update

Xheepey87 opened the pull request:
Changed some Traditional Chinese texts which are Simplified Chinese.

+ Fixed again, based on the Traditional Chinese mod
https://steamcommunity.com/sharedfiles/filedetails/?id=1111711682

linh0311 commented:
A little bit confuse: Did chinese translate those recipe before? I see a lot of changes and updates.

Xheepey87 commented:
@linh0311 
I don't know what you mean exactly.
Before I began to help Chinese translation, cheewind, one of authors of recipe book filter, was the only one who did Chinese translation, included Traditional Chinese and Simplified Chinese. But as far as I know, he is a Traditional Chinese speaker, not a Simplified Chinese speaker, so I did a lot of changes to Simplified Chinese. This is why there were several Chinese translation updates. In addition, I'm a new member of github, maybe I did something wrong that I didn't know.
After first update, I noticed there were some texts which belonged to Simplified Chinese but were at Traditional Chinese. So I updated again, and at that time I thought cheewind would notice them too and fix them, but he didn't. So I helped him.
And here is it. All of the updates for now are done by cheewind and me, not anyone more. If no special situation, it should be the last time I update Chinese translation.

ricardjorg commented:
Merged in! Thanks all!
